test/utils/TestKeyManager.sol

```diff
@@ -41,11 +41,7 @@ library TestKeyManager {
             return TestKey({keyType: keyType, publicKey: abi.encodePacked(x, y), privateKey: DEFAULT_SECP256R1_PK});
         } else if (keyType == KeyType.Secp256k1) {
             address defaultAddress = vm.addr(DEFAULT_SECP256K1_PK);
-            return TestKey({
-                keyType: keyType,
-                publicKey: abi.encodePacked(defaultAddress),
-                privateKey: DEFAULT_SECP256K1_PK
-            });
+            return TestKey({keyType: keyType, publicKey: abi.encode(defaultAddress), privateKey: DEFAULT_SECP256K1_PK});
```

Comment:
I think some files formatting is messed up, I'm gonna make a PR to move out the layout syntax so we can start to run forge fmt locally

Reply:
& nice - this should be abi.encode not abi.encodePacked for sure